lib/wor/paginate/adapters/enumerable.rb

```diff
@@ -0,0 +1,31 @@
+# Used when render_paginated is called with an Enumerable, something like
+### render_paginated [1,2,3,4]
+module Wor
+  module Paginate
+    module Adapters
+      class Enumerable < Adapter
+        attr_reader :page
+
+        def required_methods
+          [:to_a]
+        end
+
+        def paginated_content
+          return @paginated_content if @paginated_content
+          content_array = @content.to_a
+          @paginated_content = content_array.slice((page - 1) * @limit, @limit)
```

Comment:
Here we are not validating if page - 1 is negative. Probably we should move that login into a method in the base class